components/ActionSheet/src/private/MDCActionSheetHeaderView.m

```diff
@@ -122,13 +122,9 @@ - (NSString *)title {
 
 - (void)setMessage:(NSString *)message {
   self.messageLabel.text = message;
-  // If message is empty or nil then the title label's alpha value should be lighter, if there is both
-  // then the title label's alpha should be darker.
-  if (self.message && ![self.message isEqualToString:@""]) {
-    _titleLabel.textColor = [UIColor.blackColor colorWithAlphaComponent:TitleLabelAlpha];
-  } else {
-    _titleLabel.textColor = [UIColor.blackColor colorWithAlphaComponent:MessageLabelAlpha];
-  }
+  self.titleLabel.textColor = self.titleTextColor ?: [self defaultTitleTextColor];
+  self.messageLabel.textColor =
+      self.messageTextColor ?: [UIColor.blackColor colorWithAlphaComponent:MessageLabelAlpha];
```

Comment:
Consider pulling this logic out to a "updateLabelColors" call that you can invoke from `setMessage:`, `setTitleTextColor:`, and `setMessageTextColor:`.

Reply:
Done